code/game/objects/items/weapons/material/whetstone.dm

```diff
@@ -8,4 +8,75 @@
 	force = 3
 	w_class = ITEMSIZE_SMALL
 	var/repair_amount = 5
-	var/repair_time = 40
\ No newline at end of file
+	var/repair_time = 40
+
+/obj/item/weapon/whetstone/attackby(obj/item/I, mob/usr)
```

Comment:
do not use usr as a var name, that will collide with byond's built in usr var

Reply:
how many fucking var names are there that aren't listed